frontend/package.json

```diff
@@ -24,10 +24,13 @@
     "@vitejs/plugin-vue": "^5.0.5",
     "@vue/tsconfig": "^0.5.1",
     "@vuepic/vue-datepicker": "^8.8.0",
-    "axios": "^1.7.2",
+    "axios": "^1.7.7",
```

Comment:
Ideally, we're using fixed versions. Including the `~` or `^` to allow for auto-updating on build can sometimes lead to unexpected importing